web/src/components/HelpText.js

```diff
@@ -9,7 +9,7 @@ const HelpText = ({ text: textResource, reminder: reminderResource }) => {
 
   return (
     (text || reminder) && (
-      <div className="mb2" style={{ whiteSpace: 'pre-line' }}>
+      <div className="mb2">
```

Comment:
Should this have the `pre-line` class?  I *think* this was to support help text that had newline characters but I can't remember exactly.